doc/lkt_coding_style.rst

```diff
@@ -0,0 +1,81 @@
+****************
+LKT coding style
+****************
+
+This is the current - evolving - coding style for the LKT syntax. For the
+moment this is just organized as a flat list of rules.
+
+Indentation & braces
+====================
+
+Indentation is expected to be 4 spaces. The general rule is that everything
+inside of braces should be indented, opening brace should be on the line of the
+preceding logically linked token, and closing brace on its own line.
+
+.. code-block::
+
+    fun a(b: Int, c: Int): Int = {
+        b + c
+    }
+
+Casing
+======
+
+We generally follow the Python PEP8 casing convention. In short:
+
+* Type names should be capitalized, using CamelCase when there are several
+  parts in the name.
+* Variable names should be lowercase, using underscores when there are several
+  parts.
+
+Parens/brackets/colon
+=====================
+
+No space expected before parens, brackets and colons, for function
+declarations, calls, generic declarations, array accesses, variable
+declarations etc.
```

Comment:
I’ve noticed one exception that is worth mentionning: opening paren for function types:

```
foo: (A) -> B
```

Reply:
Reworded